index.js

```diff
@@ -9,13 +9,14 @@ module.exports = {
     //default theme name is 'default'
     var options = app.options['ember-prism'] || { components: [] };
 
-    //import main stylesheet
-    app.import(app.bowerDirectory + '/prism/themes/prism.css');
-
     //import theme based on options
     if (options.theme){
-      app.import(app.bowerDirectory + '/prism/themes/prism-' + options.theme + '.css');
+      // allow ability to specify no css if we want to provide our own
+      if (options.theme !== 'none'){
+        app.import(app.bowerDirectory + '/prism/themes/prism-' + options.theme + '.css');
+      } 
     } else {
+      // fall back to default theme
       app.import(app.bowerDirectory + '/prism/themes/prism.css');
```

Comment:
Sorry - I didn't really answer your question properly, just understanding now what you mean. Each theme file is self-sufficient. None of the alternative themes depend on the default prism.css.

Hope that helps!